### PR TITLE
Get rid of final PHPStan baseline messages 🎉

### PR DESCRIPTION
**Pull request by nickygerritsen**

### CODE REVIEW

**Comment by nickygerritsen** [comment]
We might want to raise the level to 7 though, since it seems to find some stuff. I will fix some stuff in a separate PR now.

File: webapp/src/Controller/API/JudgehostController.php [hunk 1]
```diff
@@ -113,6 +113,7 @@ public function getJudgehostsAction(
     /**
      * Add a new judgehost to the list of judgehosts.
      * Also restarts (and returns) unfinished judgings.
+     * @return array<array{jobid: int, submitid: int}>
```
Comment (vmcj) [inline webapp/src/Controller/API/JudgehostController.php:116]:
```suggestion
     *
     * @return array<array{jobid: int, submitid: int}>
```

File: webapp/src/Controller/API/GeneralInfoController.php [hunk 2]
```diff
@@ -163,6 +163,8 @@ public function getUserAction(): User
 
     /**
      * Get configuration variables.
+     *
+     * @return array<string, bool|int|array<string, string>|string>
```
Comment (vmcj) [inline webapp/src/Controller/API/GeneralInfoController.php:167]:
```suggestion
     * @return array<string, bool|int|string|array<string, string>>
```

File: webapp/src/Controller/API/AbstractRestController.php [hunk 3]
```diff
@@ -236,6 +150,7 @@ abstract protected function getIdField(): string;
 
     /**
      * @throws NonUniqueResultException
+     * @return array<U>
```
Comment (vmcj) [inline webapp/src/Controller/API/AbstractRestController.php:153]:
```suggestion
     * @return array<U>
     * @throws NonUniqueResultException
```

**Review by vmcj — APPROVED** [review]
CI happy... I'm happy

One file less to care about.

**Review by meisterT — APPROVED** [review]

**Pull request merged**